locale/pt-br/texts.po

```diff
@@ -57,6 +428,320 @@ msgstr "Saldo Disponível"
+
+#: src/components/TxDetailsModal.js:50 src/screens/PaymentRequestDetail.js:104
+msgid "Token"
+msgstr "Token"
+
+#: src/screens/PaymentRequestDetail.js:116 src/screens/Receive.js:29
+msgid "My Address"
+msgstr "Meu endereço"
+
+#: src/screens/PaymentRequestDetail.js:120
+msgid "Status"
+msgstr "Status"
+
+#: src/screens/PaymentRequestDetail.js:121
+msgid "Received"
+msgstr "Recebido"
+
+#: src/screens/PaymentRequestDetail.js:121
+msgid "Waiting confirmation"
+msgstr "Aguardando confirmação"
+
 #: src/screens/PinScreen.js:157
 msgid "Incorrect PIN Code. Try again."
 msgstr "PIN incorreto. Tente novamente."
+
+#: src/screens/PinScreen.js:53
+msgid "Enter your PIN Code "
+msgstr "Entre seu PIN"
+
+#: src/screens/PinScreen.js:54
+msgid "Unlock Hathor Wallet"
+msgstr "Desbloqueie sua Hathor Wallet"
+
+#: src/screens/PinScreen.js:169
+msgid "Cancel"
+msgstr "Cancelar"
+
+#: src/screens/PinScreen.js:172 src/screens/Settings.js:128
+msgid "Reset wallet"
+msgstr "Resetar a wallet"
+
+#. *
+#.      * index {number} Selected index of the tab bar
+#.      
+#.  eslint thinks routes is not used, but TabView uses it
+#.  eslint-disable-next-line react/no-unused-state
+#: src/screens/Receive.js:30
+msgid "Payment Request"
+msgstr "Requisição de Pagamento"
+
+#: src/screens/Receive.js:105
+msgid "RECEIVE"
+msgstr "RECEBER"
+
+#: src/screens/RegisterToken.js:31 src/screens/SendScanQRCode.js:67
+msgid "Manual info"
+msgstr "Digitar"
+
+#: src/screens/RegisterToken.js:46 src/screens/RegisterTokenManual.js:112
+msgid "REGISTER TOKEN"
+msgstr "REGISTRAR UM TOKEN"
+
+#: src/screens/RegisterToken.js:57
+msgid "Scan the token QR code"
+msgstr "Leia o QR code do token"
+
+#: src/screens/RegisterTokenManual.js:88
+msgid "You're going to register the following token:"
+msgstr "Você irá registrar o seguinte token:"
+
+#: src/screens/RegisterTokenManual.js:90
+msgid "Name: "
+msgstr "Nome: "
+
+#: src/screens/RegisterTokenManual.js:94
+msgid "Symbol: "
+msgstr "Símbolo: "
+
+#: src/screens/RegisterTokenManual.js:118
+msgid "Configuration string"
+msgstr "Configuração"
+
+#: src/screens/ResetWallet.js:75
+msgid "RESET WALLET"
+msgstr "RESETAR WALLET"
+
+#: src/screens/ResetWallet.js:79
+msgid "Are you sure?"
+msgstr "Você tem certeza?"
+
+#: src/screens/ResetWallet.js:82
+msgid ""
+"If you reset your wallet, **all data will be deleted**, and you will **lose "
+"access to your tokens**."
+msgstr ""
+"Se você resetar sua wallet, **todos os seus dados serão apagados**, e você "
+"irá **perder acesso aos seus tokens**."
+
+#: src/screens/ResetWallet.js:84
+msgid ""
+"To recover access to your tokens, you will need to import your seed words "
+"again."
+msgstr ""
+"Para recuperar acesso aos seus tokens, você irá precisar importar suas "
+"palavras novamente."
+
+#: src/screens/ResetWallet.js:93
+msgid ""
+"I want to reset my wallet, and I acknowledge that **all data will be wiped "
+"out**."
+msgstr ""
+"Eu quero resetar minha wallet, e eu concordo que **todos os dados serão "
```

Comment:
"eu concordo" I would change to "eu entendo" and "permanentemente" I think it's too much. What do you think?

Reply:
FIxed.